c/zosaccounts.c

```diff
@@ -266,6 +384,17 @@ int groupIdGet (char *string, int *returnCode, int *reasonCode) {
   return groupId;
 }
 
+/* Obtain group name by GID, groupNameBuffer must be at least 8 chars long */
+int groupGetName(int gid, char *groupNameBuffer, int *returnCode, int *reasonCode) {
+  int status = 0;
+  GroupInfo groupInfo = {0};
+  status = getGroupInfo(gid, &groupInfo, returnCode, reasonCode);
+  if (status == 0) {
```

Comment:
Should we check if groupNameBuffer is at least 8 chars long before getting here?

Reply:
We are unable to check if `groupNameBuffer` because the caller is responsible for allocating the buffer that is large enough.